Ownership/TC - LTC/London TC NE.txt

```diff
@@ -9,7 +9,7 @@ ALTOWNER:Observing LON_SC_CTR:TCGS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:LGS:O:LMEN:LS
 ALTOWNER:Observing London FIR:TCGS:LGS:O:LMEN:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
 
 SECTOR:SODVU Box:8500:10500
-OWNER:TCSE:TCS:TC:LD:LS:LSC:L
+OWNER:TCNE:TCN:TC:DAGGA:TCE:LEN:LE:LC:LSC:L
```

Comment:
This is for the purpose of climbing Thames deps - needs defining here because of ownership orders.

```suggestion
OWNER:TCSE:TCS:TC:LD:LS:LSC:L
```